packages/discovery-react-components/src/utils/formatMessage.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
```

Comment:
i know we had been recommending doing `import * from React` before for storybook, but I believe we don't need to do that here (or anywhere) as long as we don't mix `React.Thing` and just `Thing`

Reply:
Ah yes, this snuck back in from before. I can also add something to CONTRIBUTING and confirm we don't have any lingering elsewhere.